python/lsst/pipe/tasks/multiBand.py

```diff
@@ -353,7 +353,7 @@ class DeblendCoaddSourcesConfig(Config):
                                           doc="Deblend sources separately in each band")
     multiBandDeblend = ConfigurableField(target=ScarletDeblendTask,
                                          doc="Deblend sources simultaneously across bands")
-    simultaneous = Field(dtype=bool, default=False, doc="Simultaneously deblend all bands?")
+    simultaneous = Field(dtype=bool, default=True, doc="Simultaneously deblend all bands?")
```

Comment:
The doc for this field needs to say that this used to switch between the `singleBandDeblend` and `multiBandDeblend` subtasks listed above.